vispy/app/backends/_wx.py

```diff
@@ -218,13 +218,18 @@ def __init__(self, *args, **kwargs):
         else:
             self._gl_context = p.context.shared.ref._gl_context
 
+        if p.position == None:
```

Comment:
Flake/linting is failing on travis tests. Please change this to `if p.position is None:`.

Reply:
ok